server/__tests__/app.spec.js

```diff
@@ -129,7 +129,7 @@ describe('App', () => {
           hubFeaturedContent: jest.fn().mockRejectedValue(error),
         },
         offenderService: {
-          getEventsForToday: jest.fn().mockResolvedValue([]),
+          getEventsListForToday: jest.fn().mockResolvedValue([]),
```

Comment:
rename - this returns a list of events for the home page, where as we wanted to add a `getEventsForToday` that returned the same type as `getEventsFor`